docs/unified-test-documentation/dasharo-compatibility/326b-firmware-building-locally.md

```diff
@@ -0,0 +1,98 @@
+# Dasharo Compatibility: Firmware building locally
```

Comment:
This test suite hasn't been visible in the test documentation - please add it `mkdocs.yml` file.